### DISCUSSION TITLE
正文中表格和图片的引用如何添加括号（比如：如图(1-1)所示，...）

### DISCUSSION THREAD

**Discussion by Wilson1996** [sjtug/SJTUThesis#754] [提问] [answered]
直接在正文中加括号会有多余的空格出现，请问在模板的哪里可以修改满足这个需求呢？非常感谢

**Comment by AlexaraWu** [comment, marked as answer]
一个比较 tricky 的方法，直接用 `amsmath` 的 `\eqref`：
![image](https://user-images.githubusercontent.com/15205102/149309109-1946f2f6-b117-4a05-9f89-0adc517e4ab1.png)

**Reply by Wilson1996** [reply]
感谢大佬！

**Comment by zepinglee** [comment]
中文和西文之间本来就要加空格，这里最正确的写法为 `如图~(\ref{fig:1-1}) 所示`。

**Reply by Wilson1996** [reply]
这个我之前尝试过，但是效果是图( 1-1)这样的，左括号与数字之间有一个空格
